taxonomies/categories.txt

```diff
@@ -44294,6 +44299,14 @@ sv:Skyr
 zh:Skyr
 wikidata:en:Q742127
 
+<en:Skyr
```

Comment:
Can we put "Skyrs" in plural, and then Skyr in singular in all the related entries?

Same for Quark?

Reply:
I modified it